envi/archs/arm/disasm.py

```diff
@@ -1455,7 +1468,7 @@ def render(self, mcanv):
         daib_flags = self.iflags & IF_DAIB_MASK
         if self.iflags & IF_L:
             mnem += 'l'
-        elif self.iflags & IF_PSR_S:
+        elif self.iflags & IF_PSR_S and not (self.iflags & IF_PSR_S_SIL):
```

Comment:
this works.  i prefer the following approach (for speed):

defined in global module context:
S_FLAG_MASK = IF_PSR_S | IF_PSR_S_SIL

in the function:
    elif (self.iflag & S_FLAG_MASK) == IF_PSR_S:

the performance of the math is faster than two distinct checks.